app/scripts/modules/core/src/delivery/create/CreatePipelineModal.tsx

```diff
@@ -306,7 +306,7 @@ export class CreatePipelineModal extends React.Component<ICreatePipelineModalPro
           <Modal.Title>Create New {this.state.command.strategy ? 'Strategy' : 'Pipeline'}</Modal.Title>
         </Modal.Header>
         {this.state.loading && (
-          <Modal.Body style={{height: '200px'}}><Spinner radius={25} width={6} length={16} /></Modal.Body>
+          <Modal.Body style={{height: '200px'}}><LegacySpinner radius={25} width={6} length={16} /></Modal.Body>
```

Comment:
If you want to use the new spinners: 
React: <Spinner size="nano" />
Angular: <loading-spinner size="'nano'"></loading-spinner>

Reply:
@archana-s I will leave it for you to convert these to the new spinners. I really just wanted to get the new React Spinner available in the core library build and make it more clear that these are the old spinners and we shouldn't use them.